Looks like it still does not work due to https://github.com/python/cpython/pull/6475

```
$ pip install 'sphinx<1.6' blurb
$ make htmlhelp
mkdir -p build
Building NEWS from Misc/NEWS.d with blurb
PATH=./venv/bin:$PATH sphinx-build -b htmlhelp -d build/doctrees -D latex_elements.papersize=  . build/htmlhelp 
Running Sphinx v1.5.6

Extension error:
Could not import extension pyspecific (exception: cannot import name 'status_iterator' from 'sphinx.util' (/home/mdk/.venvs/tmp/lib/python3.7/site-packages/sphinx/util/__init__.py))
make: *** [Makefile:45: build] Error 1
```

--- a/Doc/tools/extensions/pyspecific.py
+++ b/Doc/tools/extensions/pyspecific.py
@@ -23,7 +23,6 @@ from docutils import nodes, utils
 from sphinx import addnodes
 from sphinx.builders import Builder
 from sphinx.locale import translators
-from sphinx.util import status_iterator
 from sphinx.util.nodes import split_explicit_title
 from sphinx.writers.html import HTMLTranslator
 from sphinx.writers.text import TextWriter, TextTranslator
@@ -314,6 +313,11 @@ class PydocTopicsBuilder(Builder):
         return ''  # no URIs
 
     def write(self, *ignored):
+        try:  # sphinx>=1.6
+            from sphinx.util import status_iterator
+        except ImportError:  # sphinx<1.6
+            status_iterator = self.status_iterator
+
         writer = TextWriter(self)
         for label in status_iterator(pydoc_topic_labels,
                                      'building topics... ',

